.env.example

```
@@ -54,3 +54,5 @@ POSTHOG_HOST=
 LIFECYCLE_TEST_BUCKET=
 # Secret token for authenticating lifecycle test endpoint calls
 LIFECYCLE_TEST_TOKEN=
+PROVISIONING_API_URL=
+PROVISIONING_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PROVISIONING_API_KEY key should go before the PROVISIONING_API_URL key

(UnorderedKey)